### DISCUSSION TITLE
Problem in wire:navigate in all parts in version

### DISCUSSION THREAD

**Discussion by HoseinSatari** [livewire/livewire#6810] [7. Bugs (archived)]
### Livewire version

3.0.2 and 3.0.3

### Which PHP version are you using?

PHP 8.1

### Repository URL

https://github.com/HoseinSatari/nordes.ir.git

### Steps To Reproduce

When I click on a link that uses the navigate mode, the string value 2023 is printed on the new page, and when I back it, the same value is printed again. 

### Additional information

after click on this code : 
`  <li><a href="{{ route('home') }}" wire:navigate class="sub-menu-item">صفحه اصلی </a></li>`
and then show this :
![image](https://github.com/livewire/livewire/assets/74406383/8d682025-f96d-4b8b-b318-34a27dc2dce6)

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by HoseinSatari** [comment]
please check url : http://nordes.ir/
and then click on "صفحه اصلی" in header menu

**Comment by Graphicfort** [comment]
<img src="https://i.ibb.co/GHgSBX5/Screenshot-2023-09-14-151856.png">

1- wire:navigate is missing
2- check browser console
